src/components/Book.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ title, author }) => (
+  <>
+    <h3>{title}</h3>
+    <h4>{author}</h4>
+  </>
```

Comment:
- [ ] Kindly consider adding a button here  `<button>Delete</button>` to delete the book. it will be useful for the future milestone. right now it will just act as a placeholder.